.gitignore

```diff
@@ -7,4 +7,5 @@ prebuilds/
 lib/
 libexec/
 share/
+include/
```

Comment:
Wanted to fix a small visual issue with Atom. The include folder is empty after a build so nothing is committed, but it still shows up as a normal folder in Atom.